include/OpenColorIO/OpenColorIO.h

```diff
@@ -634,6 +652,148 @@ class OCIOEXPORT Config
 
 extern OCIOEXPORT std::ostream& operator<< (std::ostream&, const Config&);
 
+//!cpp:class:: The File Rules are a set of filepath to color space mappings that are evaluated
+// from first to last. The first rule to match is what determines which color space is
+// returned. There are four types of rules available. Each rule type has a name key that may
+// be used by applications to refer to that rule. Name values must be unique. The other keys
+// depend on the rule type:
+//
+// - Basic Rule: This is the basic rule type that uses Unix glob style pattern matching and
+// is thus very easy to use. It contains the keys:
+//   * name: Name of the rule
+//   * colorspace: Color space name to be returned.
+//   * pattern: Glob pattern to be used for the main part of the name/path.
+//   * extension: Glob pattern to be used for the file extension. Note that if glob tokens
+//     are not used, the extension will be used in a non-case-sensitive way by default.
+// 
+// - Regex Rule: This is similar to the basic rule but allows additional capabilities for
+//   power-users. It contains the keys:
+//   * name: Name of the rule
+//   * colorspace: Color space name to be returned.
+//   * regex: Regular expression to be evaluated.
+// 
+// - OCIO v1 style Rule: This rule allows the use of the OCIO v1 style, where the string
+//   is searched for color space names from the config. This rule may occur 0 or 1 times
+//   in the list. The position in the list prioritizes it with respect to the other rules.
+//   It has the key:
+//   * name: Must be "ColorSpaceNamePathSearch".
+// 
+// - Default Rule: The file_rules must always end with this rule. If no prior rules match,
```

Comment:
We should mention `strict_parsing` here, noting that this rule will return an empty string when it is enabled, assuming this is still true.

Reply:
strict parsing is actually not used here. If no color space is found in the path the rule will not be triggered.

Reply:
Are we no longer honoring the previous functionality of parsing for a path returning a empty string when strict parsing is on? I thought we were trying to preserve that for those that want it.

Reply:
If you use Config::parseColorSpaceFromString nothing has changed, strictParsing will continue to work as before.
If you use FileRules ( FileRules::getColorSpaceFromFilepath ) then strictParsing has no effect.

Reply:
Great to know, thanks!